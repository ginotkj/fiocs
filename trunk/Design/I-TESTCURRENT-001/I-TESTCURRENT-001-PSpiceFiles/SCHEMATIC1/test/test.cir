** Profile: "SCHEMATIC1-test"  [ D:\Documents\TESIS\TRUNK\Design\I-TESTCURRENT-001\i-testcurrent-001-pspicefiles\schematic1\test.sim ] 

** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
.LIB "../../../i-testcurrent-001-pspicefiles/i-testcurrent-001.lib" 
* From [PSPICE NETLIST] section of C:\OrCAD\OrCAD_16.2\tools\PSpice\PSpice.ini file:
.lib "nomd.lib" 

*Analysis directives: 
.TRAN  0 1n 0.1p 0.001u 
.PROBE V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 


.END
