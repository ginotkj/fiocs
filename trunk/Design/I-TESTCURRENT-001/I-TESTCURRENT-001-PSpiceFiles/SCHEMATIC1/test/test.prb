[DISPLAYS]
BEGIN DISPLAY LAST SESSION
ANALYSIS TRANSIENT_ANALYSIS
SYMBOL ALWAYS
TRACECOLORSCHEME NORMAL
BEGIN ANAPLOT 1
ACTIVE
XBASE
BEGIN XAXIS
XAXISUSERNAME 0 User Defined Title
RANGEFLAG AUTO
TYPE LOG
UNIT s
BEGIN GRIDS
MAJORSPACINGLINEAR 2e-011
MAJORSPACINGLOG 1
MINORINTERVALS 2
MAJORNUMBERS
MAJORSTYLE LINES
MAJORPATTERN SOLID
MINORSTYLE NONE
END GRIDS
END XAXIS
BEGIN YAXIS 1
YAXISSIDE LEFT
ACTIVE
RANGEFLAG AUTO
TYPE LINEAR
UNIT A
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE LINES
MAJORPATTERN SOLID
MINORSTYLE NONE
END GRIDS
BEGIN TRACE I(I2)
MARKERID 4
TRACEADDEXT
END TRACE I(I2)
BEGIN TRACE -I(R1)
MARKERID 5
TRACEADDEXT
END TRACE -I(R1)
END YAXIS 1
END ANAPLOT 1
END DISPLAY LAST SESSION
