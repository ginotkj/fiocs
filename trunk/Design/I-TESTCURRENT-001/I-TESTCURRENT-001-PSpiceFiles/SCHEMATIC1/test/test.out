
**** 07/28/09 22:46:10 ******** PSpice Lite (Oct 2008) ******** ID# 10813 ****

 ** Profile: "SCHEMATIC1-test"  [ D:\Documents\TESIS\TRUNK\Design\I-TESTCURRENT-001\i-testcurrent-001-pspicefiles\schematic1\test.sim


 ****     CIRCUIT DESCRIPTION


******************************************************************************




** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
.LIB "../../../i-testcurrent-001-pspicefiles/i-testcurrent-001.lib" 
* From [PSPICE NETLIST] section of C:\OrCAD\OrCAD_16.2\tools\PSpice\PSpice.ini file:
.lib "nomd.lib" 

*Analysis directives: 
.TRAN  0 100p 0.1p 0.001u 
.PROBE V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 



**** INCLUDING SCHEMATIC1.net ****
* source I-TESTCURRENT-001
I_I2         N16014 0 DC 0Adc AC 0Aac
+PULSE 0 .3m 1p 15p 30p 10p 0
I_I1         N16014 0 DC 0Adc AC 0Aac
+PULSE 0 1m 1p 10p 10p 0 0
R_R2         0 ITEST  1k TC=0,0 
R_R1         N16014 ITEST  1k TC=0,0 

**** RESUMING test.cir ****
.END

**** 07/28/09 22:46:10 ******** PSpice Lite (Oct 2008) ******** ID# 10813 ****

 ** Profile: "SCHEMATIC1-test"  [ D:\Documents\TESIS\TRUNK\Design\I-TESTCURRENT-001\i-testcurrent-001-pspicefiles\schematic1\test.sim


 ****     INITIAL TRANSIENT SOLUTION       TEMPERATURE =   27.000 DEG C


******************************************************************************



 NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE


(ITEST)    0.0000  (N16014)    0.0000 




    VOLTAGE SOURCE CURRENTS
    NAME         CURRENT


    TOTAL POWER DISSIPATION   0.00E+00  WATTS



          JOB CONCLUDED

**** 07/28/09 22:46:10 ******** PSpice Lite (Oct 2008) ******** ID# 10813 ****

 ** Profile: "SCHEMATIC1-test"  [ D:\Documents\TESIS\TRUNK\Design\I-TESTCURRENT-001\i-testcurrent-001-pspicefiles\schematic1\test.sim


 ****     JOB STATISTICS SUMMARY


******************************************************************************



  Total job time (using Solver 1)   =         .02
